.backportrc.json

```diff
@@ -0,0 +1,14 @@
+// .backportrc.json
```

Comment:
nit: I know its a funny ask, but can we add a comment saying that comments are OK in this json file, for future maintainers?